SECURITY.md

```diff
@@ -1,22 +1,5 @@
 # Security Policy
 
-## Security Announcements
-
-Join the [kubernetes-security-announce] group for security and vulnerability announcements.
-
-You can also subscribe to an RSS feed of the above using [this link][kubernetes-security-announce-rss].
-
 ## Reporting a Vulnerability
 
-Instructions for reporting a vulnerability can be found on the
-[Kubernetes Security and Disclosure Information] page.
-
-## Supported Versions
-
-Information about supported Kubernetes versions can be found on the
-[Kubernetes version and version skew support policy] page on the Kubernetes website.
-
-[kubernetes-security-announce]: https://groups.google.com/forum/#!forum/kubernetes-security-announce
-[kubernetes-security-announce-rss]: https://groups.google.com/forum/feed/kubernetes-security-announce/msgs/rss_v2_0.xml?num=50
-[Kubernetes version and version skew support policy]: https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions
-[Kubernetes Security and Disclosure Information]: https://kubernetes.io/docs/reference/issues-security/security/#report-a-vulnerability
+Please report security vulnerabilities to: security@servicebinding.io
```

Comment:
I kept this PR in draft form as the email address doesn't exist. Let me know if we should use a couple of our contributor's email addresses here.

Reply:
Can we use the same emails as the code of conduct?

Reply:
Let's check with @nebhale and @arthurdm If they are fine with it. I have added a topic for discussion to our next call.

Reply:
Based on the community call discussion, we have decided to create security@servicebinding.io email address and add @scothis and @sadlerap as the recipients. I will ask Red Hat's open source program team to help on the same.

Reply:
The email address has been created.
@scothis @sadlerap Please confirm if you have received a test mail from Josh (My colleague in Red Hat).